src/dipdup/config/__init__.py

```diff
@@ -98,11 +99,13 @@ class SqliteDatabaseConfig:
     :param kind: always 'sqlite'
     :param path: Path to .sqlite file, leave default for in-memory database (`:memory:`)
     :param immune_tables: List of tables to preserve during reindexing
+    :param migrations_dir: Where to store migrations files (relative to project root)
```

Comment:
I suggest removing an ability to configure this path and hardcode it in `DipDupPackage` class instead (but don't create by default). We already use fixed directory paths for every other section; it makes things much simpler to maintain.